src/storage/table.cc

```diff
@@ -196,5 +196,15 @@ bool Table::CheckFieldExist(const std::string& name) {
     return false;
 }
 
+bool Table::CheckTsInIndex(uint32_t idx, uint32_t* ts_id) {
```

Comment:
move this function to the class of [`IndexDef`](https://github.com/4paradigm/OpenMLDB/blob/main/src/storage/schema.h#L189) as `GetTsColumnIdx`